app/models/petition.rb

```diff
@@ -220,4 +239,15 @@ def signature_counts_by_postal_district
       end
     end
   end
+
+  def validate_number_of_sponsor_emails
+    if sponsor_emails.count < AppConfig.sponsor_count_min
```

Comment:
As per my comment on the specs (see below), consider adding a validation that we're not trying to add more than ``AppConfig.sponsor_count_max``.  As it stands when someone does try to add more than that we're going to silently truncate their list and they may not realise.  It's probably better to let them choose which of the extra emails to remove rather than us just picking the first 20 of them.